app/repos/police-forces.js

```diff
@@ -13,6 +18,79 @@ const getPoliceForces = async () => {
   }
 }
 
+const addForce = async (policeForce, user, transaction) => {
+  if (!transaction) {
+    return await sequelize.transaction(async (t) => addForce(policeForce, user, t))
+  }
+
+  const findQuery = {
+    where: {
+      name: {
+        [Op.iLike]: `%${policeForce.name}%`
+      }
+    }
+  }
+  const foundPoliceForce = await sequelize.models.police_force.findOne(findQuery)
+
+  if (foundPoliceForce !== null) {
+    throw new DuplicateResourceError(`Police Force with name ${policeForce.name} already exists`)
+  }
+
+  let createdPoliceForce
+
+  const foundParanoid = await sequelize.models.police_force.findOne({
+    ...findQuery,
+    paranoid: false
```

Comment:
Do you need to add 'transaction' to keep it in the same scope? Probably not really necessary as it would have been removed on a previous transaction anyway

Reply:
left it out as the select query didn't need to be rolled back, and failure of this would cause the whole transaction to fail anyway